tool/lib/commands/release_helper.dart

```diff
@@ -23,21 +27,32 @@ class ReleaseHelperCommand extends Command {
 
   @override
   FutureOr? run() async {
+    final processManager = ProcessManager();
+
     final useCurrentBranch = argResults!['use-current-branch']!;
-    final currentBranchResult = await DevtoolsProcess.runOrThrow('git', [
-      'rev-parse',
-      '--abbrev-ref',
-      'HEAD',
-    ]);
-    final initialBranch = currentBranchResult.stdout.toString().trim();
+    final currentBranchResult = await runProcess(
+      processManager,
```

Comment:
bonus points: let's make runProcess and runAll extension methods on ProcessManager so we don't have to pass in the processManager parameter everywhere